Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
-name = "ckb-merkle-mountain-range"
+name = "polkadot-ckb-merkle-mountain-range"
 version = "0.6.0"
 authors = ["Nervos Core Dev <dev@nervos.org>"]
 edition = "2018"
 license = "MIT"
-description = "A generalized merkle mountain range implementation"
+description = "A generalized merkle mountain range implementation (polkadot fork)"
```

Comment:
Should we also change the repository below to `https://github.com/paritytech/merkle-mountain-range` ? Or should we keep the original one that we forked ? Just asking, not sure what's the procedure.

The same question for the `authors` field.

Reply:
Also not sure if from a licensing point of view this is ok. Or if we have to do something extra. @Morganamilo  Do you happen to know ?

Reply:
Yea repo should be changed i think. For the authors we can mention Nervos and Parity, since you modified it.

Reply:
> Yea repo should be changed i think. For the authors we can mention Nervos and Parity, since you modified it.

And `Robert Hambrock <roberthambrock@gmail.com>`

Reply:
License is fine upstream really should have a LICENSE file with their copyright attrib but they don't. I'll change the repo and authors.

Reply:
I added parity as an author. where does Robert come from?

Reply:
> License is fine upstream really should have a LICENSE file with their copyright attrib but they don't. I'll change the repo and authors.

Ok. Please also add `Robert Hambrock <roberthambrock@gmail.com>` to the authors. He contributed with this PR: https://github.com/paritytech/merkle-mountain-range/pull/1 which is basically our main change in the fork